kore/src/Kore/ASTVerifier/SentenceVerifier.hs

```diff
@@ -260,6 +266,31 @@ verifySymbolSentence indexedModule sentence =
     findSort = findIndexedSort indexedModule
     sortParams = (symbolParams . sentenceSymbolSymbol) sentence
 
+    verifyConstructorNotInHookedSort :: Either (Error VerifyError) ()
+    verifyConstructorNotInHookedSort =
+        let
+            symbol = symbolConstructor $ sentenceSymbolSymbol sentence
+            attributes = sentenceSymbolAttributes sentence
+            resultSort = sentenceSymbolResultSort sentence
+            resultSortId = getSortId resultSort
+
+            ctor =
```

Comment:
Please add a `TODO` here to look the attribute up in the symbol attribute record.